### PR TITLE
WIP: FEATURE `cr:import` and `cr:export commands`

### PR DESCRIPTION
**Pull request by mficzel**
All WIP for now

**Upgrade instructions**

**Review instructions**

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

Related: https://github.com/neos/neos-development-collection/issues/4087

### CODE REVIEW

File: Neos.ContentRepository.Export/src/ExportService.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+use Neos\EventStore\EventStoreInterface;
+
+class ExportService implements ContentRepositoryServiceInterface
+{
+
+    public function __construct(
+        private readonly Filesystem          $filesystem,
+        private readonly WorkspaceFinder     $workspaceFinder,
+        private readonly EventStoreInterface $eventStore,
+        private readonly ContentStreamId     $contentStreamIdentifier,
```
Comment (bwaidelich) [inline Neos.ContentRepository.Export/src/ExportService.php]: Just a cosmetic consideration:
```suggestion
        private readonly Filesystem $filesystem,
        private readonly WorkspaceFinder $workspaceFinder,
        private readonly EventStoreInterface $eventStore,
        private readonly ContentStreamId $contentStreamIdentifier,
```
Though, not just cosmetic: When aligning parameters like this, adding a new param can lead to a multi-line diff which makes reviewing harder

**Comment by skurfuerst** [comment]
@dlubitz  @mficzel Hey :) what is missing to get this merged? :) Anything I can do to help?

**Comment by dlubitz** [comment]
@skurfuerst 
ATM it just support the export/import of content repositories without assets, as the asset usage is needed for this (#4088). We could merge it anyways without the asset support and add this later. But in this case we should remove/comment out this line for now: https://github.com/neos/neos-development-collection/pull/4077/files#diff-b073f2a9c7679925ce165034722f4b2d348f73ef30ab2d597c2d27877b34a242R45

There is also this PR (#4084) open and ready to review/merge first, which adds a check on before import if the repository is empty.

Edit: And it needs reviews ;)

**Comment by skurfuerst** [comment]
Thanks @dlubitz for the overview, really helpful for me!  Then let's focus on https://github.com/neos/neos-development-collection/pull/4088 first until we get back to this one :) 

all the best, and have a great weekend,
Sebastian

**Comment by bwaidelich** [comment]
Closing in favor of #4181

**Pull request closed without merging**